src/content/docs/data-apis/ingest-apis/metric-api/cardinality-management.mdx

```diff
@@ -0,0 +1,155 @@
+---
+title: 'Cardinality management'
+metaDescription: 'You can manage your data cardinality from the New Relic UI.'
+freshnessValidatedDate: 2024-09-24
+---
+
+<Callout title="preview">
+  We're still working on this feature, but we'd love for you to try it out!
+
+  This feature is currently provided as part of a preview program pursuant to our [pre-release policies](/docs/licenses/license-information/referenced-policies/new-relic-pre-release-policy).
+</Callout>
+
+As an administrator at New Relic, one of your tasks is to make sure you don't go beyond your limit of GB Ingested. One tool for tracking your ingest is the Cardinality Management UI. You'll find helpful graphs and tables explaining your ingest so you can take action if you're approaching limits.
```

Comment:
should we spell out GB? I don't actually remember if we spell GB and MB out ever

Reply:
@jeff-colucci this is a term that's tied up in some legal language, so I just linked to the definition. How does that look?